src/style.css

```
@@ -0,0 +1,86 @@
+html {
+  color: #fff;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  display: flex;
+  line-height: 300px;
+  justify-content: center;
+  height: 300px;
+  flex-basis: 100%;
+}
+
+.block--1 {
+  background-color: rgba(255,0,0,255);
+}
+
+.block--2 {
+  background-color: rgba(204,0,0,255);
+}
+
+.block--3 {
+  background-color: rgba(153,0,0,255);
+}
+
+.block--4 {
+  background-color: rgba(102,0,0,255);
+}
+
+.block--5 {
+  background-color: rgba(51,0,0,255);
+}
+
+.block--6 {
+  background-color: rgba(0,0,0,255);
+}
+
+@media (min-width: 600px) {
+  .block {
+    flex-basis: calc(100% / 2);
+  }
+
+  .block--3 {
+    order: 2;
+  }
+
+  .block--4 {
+    order: 1;
+  }
+
+  .block--5 {
+    order: 3;
+  }
+
+  .block--6 {
+    order: 3;
+  }
```

if two or more selectors have identical styles it's a good idea to combine them

```suggestion
  .block--5,
  .block--6 {
    order: 3;
  }
```